samples/excel/89-preview-apis/pivottable-calculations.yaml

```diff
@@ -0,0 +1,217 @@
+                pivotSheet.activate();
+
+                // create the PivotTable
+                context.workbook.worksheets.getActiveWorksheet()
+                    .pivotTables.add("Farm Sales", "Data!A1:E21", "A2");
+                const pivotTable = context.workbook.worksheets.getActiveWorksheet().pivotTables.getItem("Farm Sales");
+                pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Farm"));
+                pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Type"));      
+                pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold at Farm"));
+                pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold Wholesale"));
+
+                await context.sync();
+            });
+        }
+
+        async function deletePivot() {
+            await Excel.run(async (context) => {
+                context.workbook.worksheets.getItem("Pivot").pivotTables.getItem("Farm Sales").delete();
+                
+                // also clean up the extra data from getCrateTotal()
+                context.workbook.worksheets.getActiveWorksheet().getRange("B27:C27").delete(Excel.DeleteShiftDirection.up);
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            }
+            catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }        
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to change the calculations of PivotTable data hierarchies.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+            <h3>Set up</h3>
+            <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Setup sample</span>
+            </button>
+        </section>
+
+        <section class="setup ms-font-m">
+            <h3>Change calculations</h3>
+            <button id="showPercentages" class="ms-Button">
+                <span class="ms-Button-label">Show percentages</span>
+            </button><p />
+            <button id="showDifferenceFrom" class="ms-Button">
+                <span class="ms-Button-label">Show difference from A Farms</span>
+            </button><p />
+            <button id="showSums" class="ms-Button">
+                <span class="ms-Button-label">Show sums</span>
+            </button>
+        </section>
+
+        <section class="setup ms-font-m">
+            <h3>Clean up</h3>
+            <button id="deletePivot" class="ms-Button">
+                <span class="ms-Button-label">Delete PivotTable</span>
+            </button>
+        </section>
+    language: html
+style:
+    content: |
+        /* Your style goes here */
```

Comment:
Historically, all of our snippets have used the same few lines CSS. For example, see the **range-areas** snippet that Ricky recently created:  https://github.com/OfficeDev/office-js-snippets/blob/master/samples/excel/89-preview-apis/range-areas.yaml.

I'd suggest that you add that same CSS (that's shown in Ricky's snippet) to each snippet you create for Script Lab, to ensure consistent look and feel across the various add-in task panes within Script Lab (since it looks like the HTML tags in your snippet would be impacted by the CSS).